src/smc-util/immutable-types.coffee

```diff
@@ -41,7 +40,7 @@ check_is_immutable = (props, propName, componentName="ANONYMOUS", location, prop
 allow_isRequired = (validate) ->
     check_type = (isRequired, props, propName, componentName="ANONYMOUS", location) ->
         if not props[propName]? and isRequired
-            return new Error("Required prop `#{propName}` was not specified in '#{componentName}'")
+            return new Error("Required prop `#{propName}` was not specified in `#{componentName}`")
```

Comment:
Why backticks?  Yes, for *markdown* that is right, but an error isn't markdown...;  is there an analogue to inform us in formatting based on the warnings that React produces?